packages/auth-providers/dbAuth/middleware/modules.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module '@redwoodjs/vite/middleware' {
+  export type { MiddlewareRequest } from 'r@redwoodjs/vite/middleware'
```

Comment:
Typos here.... but I need to check this with Dom, or someone working on the esm conversion.